README.md

```diff
@@ -214,6 +214,72 @@ for more information.
 
 ---
 
+---
+
+    local_receptor: false
+
+Put it to true if you want to use a local receptor bin (or download it from github) instead of package release for your distro.
```

Comment:
```suggestion
Set to true to upload a local receptor binary instead of installing via a package manager. If no local binary is present, it will be downloaded from receptor Releases on GitHub.
```